azure-pipelines.yml

```diff
@@ -24,7 +24,7 @@ pr:
 
 jobs:
 - job: Build_Native_Linux
-  timeoutInMinutes: 120
```

Comment:
should this one commit be extracted into a separate PR? (no strong opinion, just wondering if you should expedite this one while we think about the other details of this big one..)

Reply:
We could but it was only a problem, starting with the inclusion on this PR, that's why I integrated it here.